docs/integration-guides/drupal.md

```diff
@@ -64,59 +64,32 @@ import AddAppAdminConsole  from '../includes/_add-application-console.mdx';
   | **Protocol** | OIDC |
   | **Client Type** | Confidential | 
   | **PKCE** | Disabled  | 
-  | **Redirect URIs** | Use your application's App Scheme or Universal URL. This URL will also be generated for you in the OIDC module. You can always come back to change it.<br /><br />Your real redirect URI follows the pattern:<br /><br /> `https://${your-website-domain.com}/openid-connect/${client_machine_name}` | 
+  | **Redirect URIs** | This URL will also be generated for you in the OIDC module. You can always come back to change it.<br /><br />Your redirect URI follows the pattern:<br /><br /> `https://${your-website-domain.com}/openid-connect/${client_machine_name}` | 
   | **Token Endpoint Auth Method** | Client Secret Post | 
-  | **Resource Server** | None |
-  | **Grant Type** | <mark>What's recommended for this particular use case?</mark> | 
-  | **Token Format** | <mark>What's recommended for this particular use case?</mark> | 
+  | **Grant Type** | Authorization Code | 
+  | **All other options** | Use the default values for the remaining options | 
```

Comment:
As we update these tables, we should add in some of the "Why"s, like "Why Authorization Code?".

Reply:
updated :)